src/components/GraphiQL.js

```diff
@@ -895,6 +897,25 @@ function isPromise(value) {
   return typeof value === 'object' && typeof value.then === 'function';
 }
 
+function observableToPromise(observable) {
+  if ( isPromise(observable) ) {
+    return observable;
+  }
+  if ( !isObservable(observable) ) {
```

Comment:
Only the second, the first is more for readability.
If you want ill remove it..